src/utils/callGraph.ts

```diff
@@ -0,0 +1,163 @@
+import _ from 'lodash';
+import { HermesSample } from '../types/hermesProfileInterfaces';
+import { SharedEventProperties } from '../types/chromeEventInterfaces';
+import { EventsPhase } from '../types/Phases';
+export class Node {
+  children: Node[];
+  constructor(public id: number, public events: HermesSample[]) {
+    this.children = [];
+  }
+}
+
+export class CallGraph {
+  constructor(public root: Node) {}
+  /* Given an Id find the node corresponding to it */
+  findNode(id: string, node: Node): Node | undefined {
```

Comment:
This method is used to find the nodes in each graph, uses Depth First Search. We simple compare the ids of the nodes to the node we want to find.